engine/flink/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/source/flink/FlinkKafkaSource.scala

```diff
@@ -72,12 +81,63 @@ class FlinkKafkaSource[T](
 
   protected lazy val topics: NonEmptyList[TopicName.ForSource] = preparedTopics.map(_.prepared)
 
+  private val defaultOffsetResetStrategy =
+    if (kafkaConfig.forceLatestRead.contains(true)) OffsetResetStrategy.Reset else OffsetResetStrategy.Continue
+
+  override def actionParametersDefinition: Map[ScenarioActionName, Map[ParameterName, ParameterConfig]] = {
+    Map(
+      ScenarioActionName.Deploy -> Map(
+        OFFSET_RESET_STRATEGY_PARAM_NAME -> ParameterConfig(
+          defaultValue = Some(defaultOffsetResetStrategy.toString),
+          editor = Some(
+            FixedValuesParameterEditor(
+              List(
+                FixedExpressionValue(
+                  OffsetResetStrategy.Continue.toString,
+                  s"Resume reading data where it previously stopped (continue)."
+                ),
+                FixedExpressionValue(
+                  OffsetResetStrategy.Reset.toString,
+                  "Start reading new events only (reset)."
```

Comment:
"reset" and "restart" are confusing for me, both mean basically the same. From Merriam-Webster:

>reset
> : to change the reading of often to zero
> reset an odometer

> restart
> : to start anew